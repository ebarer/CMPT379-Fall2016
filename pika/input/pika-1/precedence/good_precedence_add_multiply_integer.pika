exec {

  print 93 + 7 * 42, _n_.
  print 42 + 12 * 58, _n_.
  print 88 + 20 * 20, _n_.
  print 46 + 87 * 95, _n_.
  print 52 + 18 * 15, _n_.
  print 34 + 80 * 23, _n_.
  print 81 + 11 * 4, _n_.
  print 55 + 29 * 27, _n_.
  print 30 + 4 * 2, _n_.
  print 85 + 2 * 28, _n_.
  print 34 + 17 * 92, _n_.
  print 6 + 97 * 35, _n_.

  print 60 * 66 + 58, _n_.
  print 41 * 73 + 35, _n_.
  print 98 * 45 + 43, _n_.
  print 27 * 88 + 7, _n_.
  print 3 * 74 + 42, _n_.
  print 49 * 45 + 42, _n_.
  print 75 * 45 + 24, _n_.
  print 70 * 17 + 98, _n_.
  print 42 * 67 + 71, _n_.
  print 70 * 81 + 31, _n_.
  print 43 * 97 + 81, _n_.
  print 79 * 22 + 79, _n_.

  print 66 + 92 * 80 + 52, _n_.
  print 34 + 41 * 88 + 92, _n_.
  print 22 + 22 * 11 + 17, _n_.
  print 40 + 7 * 66 + 70, _n_.
  print 33 + 86 * 96 + 28, _n_.
  print 87 + 7 * 49 + 40, _n_.
  print 18 + 42 * 30 + 22, _n_.
  print 52 + 84 * 21 + 91, _n_.
  print 63 + 93 * 99 + 93, _n_.
  print 42 + 58 * 23 + 25, _n_.
  print 21 + 71 * 37 + 22, _n_.
  print 1 + 72 * 84 + 65, _n_.

  print 69 * 14 + 13 * 52, _n_.
  print 21 * 89 + 30 * 3, _n_.
  print 95 * 60 + 62 * 64, _n_.
  print 62 * 29 + 74 * 48, _n_.
  print 59 * 73 + 30 * 80, _n_.
  print 89 * 73 + 58 * 70, _n_.
  print 42 * 35 + 85 * 41, _n_.
  print 54 * 20 + 47 * 96, _n_.
  print 24 * 10 + 31 * 79, _n_.
  print 65 * 10 + 21 * 87, _n_.
  print 42 * 88 + 38 * 27, _n_.
  print 17 * 91 + 23 * 65, _n_.

}

