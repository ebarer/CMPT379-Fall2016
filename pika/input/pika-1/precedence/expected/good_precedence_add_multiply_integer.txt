387
738
488
8311
322
1874
125
838
38
141
1598
3401
4018
3028
4453
2383
264
2247
3399
1288
2885
5701
4252
1817
7478
3734
281
572
8317
470
1300
1907
9363
1401
2670
6114
1642
1959
9668
5350
6707
10557
4955
5592
2689
2477
4722
3042
